### PR TITLE
Updates provided course run instead of canonical course run

### PR DESCRIPTION
**Pull request by jlajoie**

### CODE REVIEW

File: course_discovery/apps/publisher/dataloader/create_courses.py [hunk 1]
```diff
@@ -116,53 +116,61 @@ def transfer_course_image(meta_data_course, publisher_course):
             )
 
 
-def create_course_runs(meta_data_course, publisher_course):
-        # create or update canonical course-run for the course.
-        canonical_course_run = meta_data_course.canonical_course_run
+def create_course_runs(meta_data_course, publisher_course, course_run=None):
+        # If we have a provided course run, update that in Publisher instead of the Canonical Run
```
Comment (mikix) [inline course_discovery/apps/publisher/dataloader/create_courses.py]: Ugh, this and the old code is double indented. :(

Would have thought that would be a pylint complaint.

File: course_discovery/apps/publisher/dataloader/create_courses.py [hunk 2]
```diff
@@ -116,53 +116,61 @@ def transfer_course_image(meta_data_course, publisher_course):
...
+        else:
+            # create or update canonical course-run for the course.
+            canonical_course_run = meta_data_course.canonical_course_run
+
+            if canonical_course_run and canonical_course_run.key:
+                create_course_run(publisher_course, canonical_course_run)
+            else:
+                logger.warning(
+                    'Canonical course-run not found for metadata course [%s].', meta_data_course.uuid
+                )
```
Comment (mikix) [inline course_discovery/apps/publisher/dataloader/create_courses.py]: :+1:

**Review by mikix — APPROVED** [review]
Seems reasonable! Did not test, just scanned.

**Comment by mikix** [comment]
Tests might be nice?

File: course_discovery/apps/publisher/dataloader/create_courses.py (hunk 1 above)
Comment (jlajoie) [reply course_discovery/apps/publisher/dataloader/create_courses.py]: Fixing :(

**Pull request merged**